rocAL/source/readers/image/coco_file_source_reader.cpp

```diff
@@ -100,7 +116,7 @@ Reader::Status COCOFileSourceReader::initialize(ReaderConfig desc) {
         std::transform(file_aspect_ratio_pair.begin(), file_aspect_ratio_pair.end(), std::back_inserter(_aspect_ratios), [](auto &pair) { return pair.second; });
 
         // Copy the sorted file_names to _file_names vector to be used in sharding
-        _file_names = _sorted_file_names;
+        _all_shard_file_names_padded = _sorted_file_names;
```

Comment:
In file source reader we don't have an _all_shard_file_names_padded and it is replaced with _file_names
Can you please check that and add similar changes